src/engine/guidance/collapse_turns.cpp

```diff
@@ -463,6 +463,7 @@ RouteSteps collapseTurnInstructions(RouteSteps steps)
         if (!hasWaypointType(*previous_step))
         {
             const auto far_back_step = findPreviousTurn(previous_step);
+            const auto far_back_step_name = facade.GetNameForID(far_back_step->name_id).to_string();
```

Comment:
Unused, can this be removed, and then can the facade also be removed from the function signature?